lib/Wikia/src/Service/Helios/HeliosClientImpl.php

```diff
@@ -13,6 +13,8 @@ class HeliosClientImpl implements HeliosClient
 	const CLIENT_ID = "client_id";
 	const CLIENT_SECRET = "client_secret";
 
+	const HELIOS_AUTH_HEADER = "X-Wikia-UserId";
```

Comment:
Can we change usage of https://github.com/Wikia/app/blob/dev/lib/Wikia/src/Service/Swagger/ApiProvider.php#L10 to use this instead?